### PR TITLE
Cvvv

### PR DESCRIPTION
**Pull request by ShuryginaA**
I hope it will work)

### CODE REVIEW

File: modules/0_git/test/main.cpp [hunk 1]
```diff
@@ -7,7 +7,7 @@
 TEST(git, say_hello) {
     myspace::A a;
     EXPECT_EQ(myspace::func(a), "Hello, Nizhny!");
-    EXPECT_EQ(func(a), "Hello, World!");
+    EXPECT_EQ(func(a), "Hello, Nizhny!");
```
Comment (allnes) [inline modules/0_git/test/main.cpp:10]: Please remove task depend with git

**Comment by dkurt** [comment]
Please go to master branch **before** creating a new branch:
```
git checkout master
```
and only then
```
git checkout -b my_new_branch
```

File: modules/1_opencv/src/opencv_coins.cpp [hunk 2]
```diff
@@ -1,14 +1,49 @@
...
+    thresh.convertTo(thresh, CV_8U, 1, 0);
+    /*imshow("image", thresh);
+    waitKey();*/
+    
+    std::vector<std::vector<Point>> circuits;
+    findContours(thresh, circuits, RETR_EXTERNAL, CHAIN_APPROX_SIMPLE);
+    int sum= 0;
+    for (int i = 0; i < circuits.size(); i++)
+    {
+        double r = sqrt(contourArea(circuits[i]) / 3.14);
```
Comment (allnes) [inline modules/1_opencv/src/opencv_coins.cpp]: Please use a constant M_PI or CV_PI

File: modules/0_git/test/main.cpp (hunk 1 above)
Comment (ShuryginaA) [reply modules/0_git/test/main.cpp:10]: Explain please how

File: modules/1_opencv/src/opencv_coins.cpp (hunk 2 above)
Comment (ShuryginaA) [reply modules/1_opencv/src/opencv_coins.cpp]: Will be done

**Comment by ShuryginaA** [comment]
> Please go to master branch **before** creating a new branch:
> 
> ```
> git checkout master
> ```
> 
> and only then
> 
> ```
> git checkout -b my_new_branch
> ```

Should I do it now?

File: modules/1_opencv/src/opencv_coins.cpp (hunk 2 above)
Comment (ShuryginaA) [reply modules/1_opencv/src/opencv_coins.cpp]: Should I send a new version?

Comment (allnes) [reply modules/1_opencv/src/opencv_coins.cpp]: yes, you need to send new commit

**Comment by allnes** [comment]
> > Please go to master branch **before** creating a new branch:
> > ```
> > git checkout master
> > ```
> > 
> > 
> > and only then
> > ```
> > git checkout -b my_new_branch
> > ```
> 
> Should I do it now?

yes, its will help

**Comment by ShuryginaA** [comment]
> > > Please go to master branch **before** creating a new branch:
> > > ```
> > > git checkout master
> > > ```
> > > 
> > > 
> > > and only then
> > > ```
> > > git checkout -b my_new_branch
> > > ```
> > 
> > 
> > Should I do it now?
> 
> yes, its will help

Is it workable now?

File: modules/1_opencv/src/opencv_coins.cpp [hunk 3]
```diff
@@ -1,14 +1,50 @@
...
+       
+        double r = sqrt(contourArea(circuits[i]) / CV_PI);
+        if (r > 12)
+            sum += 2;
+        else
+            sum += 1;
+    }
+
+    return sum;
+
```
Comment (allnes) [inline modules/1_opencv/src/opencv_coins.cpp:47]: please remove redundant strings

File: modules/0_git/test/main.cpp (hunk 1 above)
Comment (allnes) [reply modules/0_git/test/main.cpp:10]: please create new branch isn't depend with old task

**Comment by dkurt** [comment]
see https://github.com/dkurt/openvino_practice/pull/106

**Pull request closed without merging**